app/views/layouts/application.html.erb

```diff
@@ -6,6 +6,9 @@
   <% end %>
 
   <% if ENV["GOOGLE_TAG_MANAGER_ID"] %>
+    <script>
+      dataLayer = [{ 'dimension1': <%= current_user.organisation_slug %> }];
```

Comment:
The developer guide suggests we use meaningful keys for the data layer keys. Is there a particular reason for calling it 'dimension1'?

https://developers.google.com/tag-manager/devguide#adding-data-layer-variables-to-a-page